data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1496 @@
+
+
+event "enlistment begins"
+	date 18 7 3014
+
+mission "Republic Bounty 1"
+	name "Republic Bounty Hunting"
+	description `Hunt down the <npc>, a pirate ship that has been assaulting miners in the Boral system.`
+	minor
+	source
+		government "Republic"
+		attributes "dirt belt"
+	to offer
+		has "event: war begins"
+		"combat rating" > 40
+		not "chosen sides"
+		random < 65
+
+	on offer
+		conversation
+			`You barely have time to find your bearings in this spaceport when you're approached by a man in a crisp, navy blue Republic uniform.`
+			`	"You have a face that says you're looking for work," he says, in a surprisingly cheerful tone. "We're in need of someone to deal some pirates that have been causing trouble near Boral. Not normally something we'd contract out, but, luckily for you, we're a little short on free ships right now."`
+			choice
+				`	"Sure."`
+				`	"Sorry, I'm not really into bounty hunting."`
+					defer
+			`	"Much obliged," he says. "It's a ship called the <npc>. It's a small little devil, but don't let it take you off guard."`
+				accept
+	on accept
+		log `Hunting a pirate ship called the "<npc>," which has been plundering ships near Boral.`
+
+	npc kill
+		personality heroic staying nemesis target
+		government Bounty
+		system
+			near Boral 1 2
+		ship "Clipper (Heavy)" "Black Fang"
+		dialog `The <npc> has been eliminated. You can claim the bounty payment by returning to <destination>.`
+
+	on complete
+		payment 175000
+		dialog `Upon landing, the commander greets you, seemingly surprised by your return. "The Republic needs more captains like you. Hopefully this payment of <payment> will be enough to convince you to help us out the next time." You nod. "Captain <last>, correct?" he says, and barely gives you enough time to agree before he continues speaking. "Right. I'm Commander Kent, in case I didn't introduce myself. I'll be keeping an eye out for you in case we ever need more assistance." `
+
+
+
+mission "Republic Bounty 2"
+	name "Republic Bounty Hunting"
+	description `Kill a known criminal aboard the <npc>, which was last spotted in the Alnasl system.`
+	source
+		government "Republic"
+		attributes "dirt belt"
+	to offer
+		has "Republic Bounty 1: done"
+		not "chosen sides"
+		random < 70
+
+	on offer
+		conversation
+			`Before you can decide where you want to go in the spaceport, you are flagged down by a Republic officer. This time, they're a dour woman with black hair in a tight bun. "Captain <last>?" she asks. You nod. "I'm Captain Shah. Commander Kent has you listed as a privateer." she says in a quick, clipped tone. "We've been made aware that a criminal has been spotted near Alnasl. He is wanted dead."`
```

Comment:
If this is on a different world it's more likely that the system simply flagged an ID, she may not have known or needed to check who listed you.
```suggestion
			`Before you can decide where you want to go in the spaceport, you are flagged down by a Republic officer. This time, they're a dour woman with black hair in a tight bun. "Captain <last>?" she asks.`
			choice
						`			"That's me."`
			`"Good, good. I'm Captain Shah. I've been keeping an eye out for someone to deal with a little problem for us." she says in a quick, clipped tone. "We've been made aware that a notable criminal has been spotted near Alnasl and our system flagged you for being listed as a privateer. He is wanted dead."`
```